File: dleyva9.c
// Danya Leyva
// lab-09

#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main(void)
{
    srand(time(NULL));

    int i = 0;
    int guess = 0;

    int answer = rand() % 20 + 1;
    // random number 1-20

    printf("Guess a number between 1 and 20:\n");
    scanf("%d", &guess);

    while (guess < 1 || guess > 20)
    {
        printf("You input a number that is out of range (1-20) re-enter a valid number.\n");
        scanf("%d", &guess);
    }

    if (guess == answer)
    {
        for (i = 0; i < guess; i++)
        {
            printf("YOU WON!!");
	    printf("you guessed the number correctly!\n");
        }
    }
    else
    {
        if (guess < answer)
        {
            printf("YOU LOSE!!");
            printf("your guess is LOWER  than my number which was %d.\n", answer);
        }
        else
        {
	    printf("YOU LOSE!!");	
            printf("your  guess is HIGHER than my number which was %d.\n", answer);
        }
    }

    return EXIT_SUCCESS;
}

File: dleyva9.java
//Danya Leyva
//Lab-09

import java.util.Scanner;
import java.util.Random;

public class dleyva9{
       public static void main(String[] args)
       {
	    Random random = new Random();
	    Scanner keyboard = new Scanner(System.in);

	    //between 10 and 20
	    int answer = random.nextInt(20)+1; 					       
            int guess =0;
	    
	    
	    System.out.println("Guess a number between 1 and 20: ");
            guess = keyboard.nextInt();

            while( guess < 1 || guess > 20 )
	    {
		    System.out.print("Your input is an number out of range(1-20) re-enter a valid number");
		    guess = keyboard.nextInt();
	    }


            if ( guess == answer)
	    { 
		    for(int i = 0; i < guess; i++)
                    {
			    System.out.println("YOU WINN!!!");
                            System.out.println("You guessed the number correctly! "+ answer + ".");
                    }
	    }	
	    else
	    {
		    if(guess <  answer)
		    {
			    System.out.println("YOU LOSE.");
			    System.out.println("Your guess is LOWER than the number "+ answer + "." );
			    
		    }
		    else 
		    {
			    System.out.println("YOU LOSE.");
			    System.out.println("Your guess is HIGHER than the number "+ answer + ".");
		    }
	    }

	    keyboard.close();
        }

}
